packages/shared/src/utils.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
The `shared` package is mostly utils anyway. Just place this in `index.ts`.

Reply:
Since `makeMap` function is also used in other files in the `shared` package, we will have to change the exports from the top of `index.ts` to at the end (if we place `makeMap` in `index.ts`).
Another approach could be to include all the helper functions in `utils.ts` and re-export in `index.ts`. This way `index.ts` will only be there for re-exporting.

Reply:
This is how that change would look like
```js
// index.ts
export * from './utils'
export * from './patchFlags'
export * from './element'
export * from './globalsWhitelist'

```

Reply:
Then maybe rename this file to simply `makeMap.ts`.

Reply:
Got it!

Reply:
Should we make the `makeMap` function as a default export in this case?